website/docs/r/dnat.html.markdown

```diff
@@ -49,5 +52,12 @@ The following arguments are supported:
 * `internal_ip` - (Required) The IP of the VM to map to
 * `protocol` - (Optional; *v2.0+*) The protocol type. Possible values are TCP, UDP, TCPUDP, ICMP, ANY. TCP is default to be backward compatible with previous version
 * `icmp_sub_type` - (Optional; *v2.0+*) The name of ICMP type. Possible values are   address-mask-request, destination-unreachable, echo-request, echo-reply, parameter-problem, redirect, router-advertisement, router-solicitation, source-quench, time-exceeded, timestamp-request, timestamp-reply, any
+* `network_type` - (Optional; *v2.4+*) Type of the network on which to apply the NAT rule. *`network_type` will be a required field in the next major version.*
+* `network_name` - (Optional; *v2.4+*) The name of the network on which to apply the SNAT. *`network_name` will be a required field in the next major version.*
 * `org` - (Optional; *v2.0+*) The name of organization to use, optional if defined at provider level. Useful when connected as sysadmin working across different organisations
 * `vdc` - (Optional; *v2.0+*) The name of VDC to use, optional if defined at provider level
+
+## NOTE
+
+When advanced edge gateway is used and rule is updated using UI, then Id mapping will be lost 
```

Comment:
I'd find explicit note that it will be removed from statefile useful. Would also put it as the other notes we put in the top of the page as seldom someone scrolls down to the bottom :)

Reply:
updated